src/xrpld/consensus/DisputedTx.h

```diff
@@ -226,8 +226,10 @@ DisputedTx<Tx_t, NodeID_t>::updateVote(
             newPosition = weight > p.avMID_CONSENSUS_PCT;
         else if (percentTime < p.avSTUCK_CONSENSUS_TIME)
             newPosition = weight > p.avLATE_CONSENSUS_PCT;
-        else
+        else if (percentTime < p.avFROZEN_CONSENSUS_TIME)
             newPosition = weight > p.avSTUCK_CONSENSUS_PCT;
+        else
+            newPosition = false;
```

Comment:
This is so simple that it's obviously correct.

Reply:
This has been rewritten a bit

Reply:
Still, the ending `newPosition = false;` remained, and I like that.